### DISCUSSION TITLE
Feature Request : add the option to set the profile status in the app

### DISCUSSION THREAD

**Discussion by P0isonousPurple** [community/community#3811] [Mobile] [answered]
I think that it would be great if you would be able to change the users status in the GitHub app, because I would like to keep people informed if I'm busy right now or not

**Comment by eliperkins** [comment, marked as answer]
Thanks for the feedback @P0isonousPurple!

Changing user status is on our radar, but we don't have a timeline for this feature currently. I've added your notes to our internal issue about this. Stay tuned!

**Comment by GuillemRoca** [comment]
FYI this was already implemented as mentioned in this other community post: https://github.com/orgs/community/discussions/113602#discussioncomment-14524006 👏🏻
